### ISSUE TITLE
for aliases for ff44

### ISSUE THREAD

**Issue by ApeKattQuest-MonkeyPython** [loujine/musicbrainz-scripts#55]
Hi, this is for https://github.com/loujine/musicbrainz-scripts/tree/aliases_for_ff44 

I've got some instruments + new ones added since (MBS-12527) that I'd like to have added to the drop down for language selection in the batch aliases script

.arz  (arb in egypt add) (basically the arabic egypt sub option)
.chr (cherokee add)
.gv (manx add)
.mai (maithili add)
.no (general norwegian add)
.rm (romansh add)
.sa (sanskrit add)
.sc (sardinian add)
.su (sundanese add)

**Comment by ApeKattQuest-MonkeyPython** [comment]
dammit, I can't seem to edit my comment. well here is the list without leading dots:

arz (arb in egypt add) (basically the arabic egypt sub option)
chr (cherokee add)
gv (manx add)
mai (maithili add)
no (general norwegian add)
rm (romansh add)
sa (sanskrit add)
sc (sardinian add)
su (sundanese add)
 (sundanese latin)
tt (tatar add)

**Comment by ApeKattQuest-MonkeyPython** [comment]
ops. can you also, for Sundanese, also add the "Sundanese Latin" option? Thank you.

**Comment by CatQuest** [comment]
in addition 

tt (tatar add)

**Comment by loujine** [comment]
Sorry for being late, I didn't have time for MusicBrainz in the last months.

Can you test the new version?
Also are you sure for the egyptian Arabic code? I would expect "ar_EG" instead

**Comment by ApeKattQuest-MonkeyPython** [comment]
Hi! didn't see this reply for some reason! 
I tested it now, and it works! The new entries in the list are not listed among the others alphabetically however, only added after chinese (zh..)

As for the egyptian arabic code I don't know what MusicBrainz has. I based it on what wikidata/wikimedia uses (eg here https://www.wikidata.org/wiki/Q5994 )

So for the internals of the list use whatever mb uses of course.

Hilariously I have a new list for you now, because the locales have been updated again:

an Aragonese
ba Bashkir
Taraskievica (no idea what mb uses here, wikipedia has "be-tarask.") (it's Belarusian Taraskievica orthography sub option on Belarusian )
co Corsican
cv Chuvash
gn Guarani
io Ido
la Latin
lij Ligurian
lmo Lombard
oc Occitan
scn Sicilian
sdh Southern Kurdish
sma Southern Sami
smj Lule Sami
sms Skolt Sami
slz Silesian
tpi Tok Pisin
vec Venetian
wa Wallon



Thank you for being patient, and sorry for not noticing this update earlier!
